package.json

```diff
@@ -15,7 +15,7 @@
   },
   "main": "./lib",
   "dependencies": {
-    "aws-sdk": "2.388.0",
+    "aws-sdk": "^2.1166.0",
```

Comment:
Do we really want to downgrade the version, and add `^`? 
However the latest `2.x.x` version will be installed because of `^`, right?